test/unit/fews/icons.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {PiWebserviceProvider} from '../../../src/piWebserviceProvider.js'
+
+import expectedResponse from '../mock/version.json'
+import 'cross-fetch/polyfill';
+import fetchMock from "fetch-mock";
+
+describe("icon resources", function () {
+    afterAll(function () {
+        fetchMock.restore();
+    });
+
+    it("get icon resource", async function () {
+        const provider = new PiWebserviceProvider("https://mock.dev/fewswebservices")
+        const results = await provider.resourcesIconsUrl('meteo_site_data.gif')
+        expect(results.toString()).toStrictEqual("https://mock.dev/fewswebservices/rest/fewspiservice/v1/resources/icons/meteo_site_data.gif");
+
```

Comment:
Split in separate `it` calls with relevant description.

Reply:
Fixed in https://github.com/Deltares/fews-pi-requests/pull/140/commits/a1a1e89e075b93ab87bf7f69583d91db3e05ec9e.